Src/Sarif.PatternMatcher.Cli/StressOptions.cs

```diff
@@ -26,24 +34,22 @@ internal class StressOptions : SingleFileOptionsBase
         public int Iterations { get; set; }
 
         [Option(
-            'd',
-            "search-definitions",
+            'p',
+            "plugin",
             Separator = ';',
             Required = true,
             HelpText = "A path to a file containing one or more search definitions to drive analysis.")]
         public IEnumerable<string> SearchDefinitionsPaths { get; set; }
 
         [Option(
-            "csv",
-            HelpText = "A path to create a csv file for outputs.")]
+            "csv-per-file",
+            HelpText = "CSV path to store analysis results per-file.")]
+        public string CSVPathPerFile { get; set; }
 
-        public string CSVFilePath { get; set; }
-    }
+        [Option(
+            "csv-aggregate",
```

Comment:
This CSV file will store aggregate analysis (like mean analysis rate, mean CPU utilization, etc). Since this is an aggregated set of results, it will append to the file, if the file exists.